index.rst

```diff
@@ -48,11 +48,256 @@
+  - Azimuth dependency. The corresponding force component are ``evt_azimuthForces.axial`` and ``evt_azimuthForces.tangent``. (To be implemented)
+  - Temperature dependency. The corresponding force component are ``evt_thermalForces.axial`` and ``evt_thermalForces.tangent``.
+  - Static bending forces. The corresponding force component is ``evt_staticForces.axial`` and ``evt_staticForces.tangent``. In Harris' lauguage, this includes `F0 <https://github.com/lsst-ts/ts_mtm2_cell/blob/master/configuration/lsst-m2/config/parameter_files/luts/FinalOpticalLUTs/F_0.csv/>`__ and `FF <https://github.com/lsst-ts/ts_mtm2_cell/blob/master/configuration/lsst-m2/config/parameter_files/luts/FinalOpticalLUTs/F_F.csv/>`__.
+  - Load cell zero offset. The corresponding force component is ``evt_loadCellZeroOffset``. To be implemented. See `Sec. 10 of SITCOMTN-004 <https://sitcomtn-004.lsst.io/#load-cell-zero-offsets/>`__.
+
+The command ``cmd_applyForces`` is not only intended for closed-loop AOS. It is also supposed to be used for manual AOS test, and bump test, where an arbituary set of forces need to be applied.
+
+There are no forces for slewing, because M2 slews passively. The actuators are much stiffer than M1M3. When the TMA slews, the M2 hard points will sense the load and the FB system will offload the forces and moments.
+
+The forces, as measured by the load cells, are given as telemetry under ``tel_forceActuatorData.axial`` and ``tel_forceActuatorData.tangent``.
+
+The M2 LUT forces are on whenever the mirror control system is in the enabled state.
+By default, the elevation angle used by the elevation dependent component of the LUT is from the onboard inclinometer.
+Only when the elevation angle from the onboard inclinometer is not available would the TMA telemetry be used.
+The source for the elevation angle can be switched between the onboard inclinometer and the TMA telemetry using the
+command ``cmd_selectElevationSource``, which takes one input parameter - 1=Onboard, 2=MTMount.
+The change in the telemetry source triggers the event ``evt_elevationSource``.
+
+######################
+M2 and Camera Hexapods
+######################
+
+The M2 and Camera hexapods have identical control interfaces. The Camera hexapod has ID=1 and M2 hexapod has ID=2.
+
+What the hexapods do is to position themselves to help acheive optimal image quality. So our discussion focuses on how we command the hexapods to the desired positions. Unlike for the mirror systems, whose LUTs have to reside with their own control systems due to safety considerations, the LUTs of the hexapods do not have to be with the hexapod CSCs. We choose to have the hexapod LUTs to be part of their owns CSCs for consistency across the AOS.
+
+The actual target position of a hexapod is the sum of two things -
+
+#. The LUT compensation
+#. The user-commanded motion
+
```

Comment:
How do we keep the offsets measured by the laser tracker? it's different from the LUT...